javascript-local-storage/data.js

```diff
@@ -0,0 +1,5 @@
+/* exported todos */
+
+var todos = [];
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+todos = JSON.parse(previousTodosJSON);
```

Comment:
Where is your `beforeunload` event listener? This looks like you didn't complete the exercise.